AFQ/tasks/segmentation.py

```diff
@@ -83,7 +83,7 @@ def segment(dwi, data_imap, mapping_imap,
 
 
 @pimms.calc("clean_bundles")
-@as_file('-clean_tractography.trk', include_track=True, include_seg=True)
+@as_file('-cleanTractography.trk', include_track=True, include_seg=True)
```

Comment:
`_tractography` will eventually be a recognized BIDS suffix, so I think this should be OK without the change. Same is true for `_xform`.